bin/gaffer

```diff
@@ -220,6 +220,7 @@ if [[ $APPLESEED ]] ; then
 		prependToPath "$appleseedPython" PYTHONPATH
 	done
 
+	prependToPath "$APPLESEED/shaders/gaffer" OSL_SHADER_PATHS
```

Comment:
I am not sure if the legacy shaders are still in the repository.

Reply:
Hmm. Perhaps I got lucky because my new Appleseed install went over the old one? I'll look into it. If that's the case though, I'm a bit concerned about breaking people's scenes - have users of your other DCC integrations had any complaints?